development/native-lxd-stack/scripts/config-ds-cs-ss.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+set -e
+
+ensure_file_exists() {
+  if [ ! -f "/etc/xroad/conf.d/$1" ]; then
+    echo "Creating /etc/xroad/conf.d/$1"
+    echo "edc:" > /etc/xroad/conf.d/"$1"
+  fi
+}
+
+configure_cs_edc_signing_key() {
```

Comment:
why not use roles/init-dev-configuration ansible? You could also use template and pass same values as here